anaconda-pkg-build/linux/alma/Dockerfile

```
@@ -0,0 +1,116 @@
+# Dockerfile for container images that Anaconda, Inc. uses to build conda
+# packages released on the "defaults" (repo.anaconda.com) channels.
+
+ARG GCC_VER=11.2.0
+
+FROM almalinux:8.10-20240528@sha256:e87da8566de5f0a10e0e6acdb7ca43da4ef97a114130691d7dd737d6923c863d
```

```suggestion
FROM almalinux:8.10-20240923@sha256:d7dbaf57916185b2be09e1eaa1156b543f3937164ffa08d7fdc020a0a3800a5a
```
This is the latest in the 8 series.